lib/mayaUsd/fileio/shading/shadingModeDisplayColor.cpp

```diff
@@ -114,9 +135,9 @@ class DisplayColorShadingModeExporter : public UsdMayaShadingModeExporter
         // (of the common graycale conversion) on re-import
         // The average is compute from the Maya color as is
         VtFloatArray displayOpacityAry;
-        const float transparencyAvg = (mayaTransparency[0] +
-                                       mayaTransparency[1] +
-                                       mayaTransparency[2]) / 3.0f;
+        const float transparencyAvg = (transparency[0] +
+                                       transparency[1] +
+                                       transparency[2]) / 3.0f;
```

Comment:
Just wanted to flag here that this particular part of the change introduced a subtle "regression" from the old behavior. This is now using the linearized `transparency` from the lambert shading node rather than the previously un-linearized `mayaTransparency`.

I opened #751 which restores the previous behavior, but as I note there, this shading mode uses this value in a somewhat bizarre way. I don't expect it would have any practical impact on any what exporting materials using the `displayColors` shading mode.